src/main/java/org/json/simple/parser/Yytoken.java

```
@@ -52,6 +52,8 @@ public String toString(){
 		case TYPE_EOF:
 			sb.append("END OF FILE");
 			break;
+		default:
++			console.log('Unknown type');
```

Please throw error instead of logging!